jib-core/src/main/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslator.java

```diff
@@ -196,5 +202,25 @@
     return ports.build();
   }
 
+  /**
+   * Converts a map of volumes strings to a list of {@link AbsoluteUnixPath}s * (e.g. {@code {@code
+   * {"/var/log/my-app-logs":{}}} -> AbsoluteUnixPath().get("/var/log/my-app-logs")}).
+   *
+   * @param volumeMap the map to convert
+   * @return a list of {@link AbsoluteUnixPath}s
+   */
+  @VisibleForTesting
+  static ImmutableList<AbsoluteUnixPath> volumeMapToList(
+      @Nullable Map<String, Map<?, ?>> volumeMap) {
+    if (volumeMap == null) {
+      return ImmutableList.of();
+    }
+    return volumeMap
+        .keySet()
+        .stream()
+        .map(AbsoluteUnixPath::get)
```

Comment:
I think we might want to catch the `IllegalArgumentException` thrown by `AbsoluteUnixPath.get` and wrap that with a `BadContainerConfigurationFormatException` like how `portMapToList` does it.

Reply:
I've added this verification and some unit tests.